events/wc_offmap_events.txt

```diff
@@ -659,12 +654,7 @@ character_event = {
 	option = {
 		name = EVTOPTA_WCOME.31
 		hidden_tooltip = {
```

Comment:
It would be nice if this effect were not hidden.

Reply:
you are not supposed to even have this modifer when using decision so it's better that way